canonical-web-and-design/ubuntu.com
# Add redirect for Openstack to openstack

## Done
There are LinkedIn posts to /Openstack which are hitting a 404.

## QA

- Open the demo or run locally
- Go to /Openstack and see it redirects
